Makefile

```diff
@@ -2,7 +2,7 @@ PROJECT = contour
 REGISTRY ?= gcr.io/heptio-images
 IMAGE := $(REGISTRY)/$(PROJECT)
 SRCDIRS := ./cmd ./internal ./apis
-PKGS := $(shell go list -mod=readonly ./cmd/... ./internal/...)
+PKGS := $(shell GO111MODULE=on go list -mod=readonly ./cmd/... ./internal/...)
```

Comment:
This shouldn't be needed if you're running Go 1.12 or later. What error did this fix?